dali/sasha/sadbghk.cpp

```diff
@@ -0,0 +1,168 @@
+#include "platform.h"
+
+#include "jlib.hpp"
+#include "jiface.hpp"
+#include "jptree.hpp"
+#include "jstring.hpp"
+
+#include "dadfs.hpp"
+#include "dalienv.hpp"
+
+#include "saserver.hpp"
+#include "sautil.hpp"
+
+static constexpr unsigned defaultHouseKeepingIntervalHours = 24;
+
+#define LOGDBGHK "DEBUGPLANEHOUSEKEEPING: "
+
+// Debug Housekeeping monitor
+
+class CSashaDebugPlaneHousekeepingServer : public ISashaServer, public Thread
+{
+    std::atomic<bool> stopped{false};
+    Semaphore stopSem;
+    Mutex runMutex;
+    StringBuffer debugDir;
+    unsigned expiryDays{0};
```

Comment:
let's add a static constexpr unsigned defaultExpiryDays = 14; and use here and on line 40 too.
And in start (before Thread::start) check that if set to 0, then bails out (doesn't start).

In fact the "Failed to get default debug plane" bail out should before the Thread::start(false) too.
Move the Thread::start(false) to the end of the function.